man/clr-power-tweaks.conf.5.md

```diff
@@ -0,0 +1,65 @@
+## SYNOPSIS
+
+**`clr-power-tweaks.conf`** is an optional configuration file for the
+clr_power(1) utility.
+
+`/etc/clr-power-tweaks.conf`
+
+## DESCRIPTION
+
+**`clr-power-tweaks.conf`** is an optional configuration file for the
+clr_power(1) utility. The configuration file contains kernel and device
+parameter values used by the clr_power(1) utility. 
+
+Configuration parameter values placed in **`clr-power-tweaks.conf`** will
+supersede the built-in defaults configured by clr_power(1). New parameters can
+be added to the configuration to supplement the built-in values. Built-in
+parameters can be changed by defining them in **`clr-power-tweaks.conf`** with
+a different value.
+
```

Comment:
This is missing the "don't touch" option that it has. If there is no value, then clr_power will not touch the file even if it has a builtin value for it. E.g. /proc/sys/vm/swappiness I believe is set to something like `30` by the kernel default. clr-power changes it to `10`. You can disable clr-power from touching the file entirely by listing the file without a value.

Reply:
This is still an open - you have "supersede" and "add" but not described the "prevent modification of parameters" covered.

Reply:
I added it to the FILE FORMAT section:

> Any parameter listed in the configuration file without a value will be excluded by clr_power(1) and the parameter will inherit the kernel's built-in default parameter value.

but now that I'm looking at it again, it makes more sense here.